### ISSUE TITLE
Add idle mode and max temp of videocard

### ISSUE THREAD

**Issue by Norbery** [RainbowMiner/RainbowMiner#471]
Hi, how about to add 2 new options:

1 - mine when pc is idle
2 - set maximum temp of videocard

**Comment by RainbowMiner** [comment]
Yes, very good! The max. temp of Nvidia video cards can already be controlled, using the build-in ocprofiles.

**Comment by Yamanipanuchi** [comment]
I second the Idle mode idea. I do not have a large need for it.. But I do have a few special cases for it.
